<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c2bExportDialog</class>
 <widget class="QDialog" name="c2bExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BibTeX File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="c2bLineEdit" name="ExportFile">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ExportFileDir">
          <property name="statusTip">
           <string>Select BibTeX directory</string>
          </property>
          <property name="icon">
           <iconset resource="cb2bib.qrc">
            <normaloff>:/icons/icons/mimeFolder.png</normaloff>:/icons/icons/mimeFolder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CopyDocuments">
        <property name="text">
         <string>Copy Documents</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="DocumentsDirectoryBox">
     <property name="title">
      <string>Documents Directory</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="c2bLineEdit" name="DocumentsDirectory">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="DocumentsDirectoryDir">
          <property name="statusTip">
           <string>Select PDF directory</string>
          </property>
          <property name="icon">
           <iconset resource="cb2bib.qrc">
            <normaloff>:/icons/icons/mimeFolder.png</normaloff>:/icons/icons/mimeFolder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="RelativeDocumentDirectory">
        <property name="text">
         <string>Set directory relative to the BibTeX File Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>c2bLineEdit</class>
   <extends>QLineEdit</extends>
   <header>c2bLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cb2bib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>c2bExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CopyDocuments</sender>
   <signal>clicked(bool)</signal>
   <receiver>DocumentsDirectoryBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
